incubator/kryo/src/test/java/org/glassfish/jersey/kryo/KryoContextResolver.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015, 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
shouldn't it be 2015, 2020?

Reply:
No, this is a new file. Git thinks the file was renamed and changed, but the previous file was erased and the new file created. The files are completely different, with different purpose